core/services/vrf/integration_v2_test.go

```diff
@@ -886,7 +1167,7 @@ func TestRequestCost(t *testing.T) {
 	t.Log(estimate)
 	// V2 should be at least (87000-134000)/134000 = 35% cheaper
 	// Note that a second call drops further to 68998 gas, but would also drop in V1.
-	assert.Less(t, estimate, uint64(85000),
+	assert.Less(t, estimate, uint64(90_000),
```

Comment:
did this actually increase above 85k? I don't see why it should have

Reply:
I can check again. I think it might be the additional `require` that increases gas used in this specific contract.

Reply:
Yeah, looks like the new estimate is 87270. Not much more than 85K. I wonder if that's all the extra code that needs to get generated for the `require` to work.